### ISSUE TITLE
badly prefixed symbols

### ISSUE THREAD

**Issue by dascritch** [mmuman/RadioTimer#1]
You must use uxFE0E accentuation before any sombol for avoiding stupid Safari behaviour.
Also, some symbols doesn't seem safe

**Comment by dascritch** [comment]
precisions about bad symbols  : 
```
	btn_item_play:"⏵"
	btn_show_titles:"🗏",
	btn_do_print: "🖶",
	glyph_speech_bubble: "🗩"
```

**Comment by mmuman** [comment]
Right, I used U+FE00 for the play button but those were added afterward, and I did't have Safari to test. 
(you mean, after, cf. https://en.wikipedia.org/wiki/Variant_form_%28Unicode%29 )
Maybe I should just force a webfont for those anyway...

**Comment by mmuman** [comment]
I tried using U+FE0E but it doesn't seem to work...
<img width="1392" alt="radiotimer" src="https://user-images.githubusercontent.com/470143/28425757-89a471f2-6d71-11e7-9206-c2e68c5a9997.png">

**Comment by mmuman** [comment]
It does work on Firefox though.
